lib/Runtime/Library/ArgumentsObject.h

```diff
@@ -122,7 +127,9 @@ namespace Js
         BOOL IsFormalArgument(PropertyId propertyId);
         BOOL IsFormalArgument(PropertyId propertyId, uint32* pIndex);    // Checks whether property is numeric, and on success sets that index.
         BOOL IsArgumentDeleted(uint32 index) const;
+    public:
         const ActivationObject* const GetFrameObject() { return frameObject; }
+        void SetFrameObject(ActivationObject * value) { frameObject = value; }
```

Comment:
nit : it is more readable if you do frameObject == nullptr

---
In reply to: [67801971](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67801971) [](ancestors = 67801971,67771245)